airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/ConfigRepositoryE2EReadWriteTest.java

```diff
@@ -194,6 +195,15 @@ void testSimpleInsertActorCatalog() throws IOException, JsonValidationException,
     assertEquals(1, catalogDbEntry);
   }
 
+  @Test
```

Comment:
how did you know to find the test file `airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/ConfigRepositoryE2EReadWriteTest.java` 

seeing it, it clearly seems like the right place, but I was less certain before you wrote a test here